mix.exs

```diff
@@ -51,7 +51,7 @@ defmodule Ret.Mixfile do
       {:httpoison, "~> 1.5"},
       {:poison, "~> 3.1"},
       {:ecto_autoslug_field, "~> 2.0"},
-      {:cors_plug, "~> 2.0"},
+      {:cors_plug, "~> 2.0.2"},
```

Comment:
cors_plug 2.01 added the ability to make `origin` a function, so I had to upgrade it.